public/const.h

```diff
@@ -355,9 +355,12 @@ enum RenderFx_t : unsigned char
 
 enum Collision_Group_t
 {
-	COLLISION_GROUP_NONE  = 0,
-	COLLISION_GROUP_DEBRIS,			// Collides with nothing but world and static stuff
-	COLLISION_GROUP_DEBRIS_TRIGGER, // Same as debris, but hits triggers
+	COLLISION_GROUP_UNKNOWN0 = 0,
+	COLLISION_GROUP_UNKNOWN1,
+	COLLISION_GROUP_UNKNOWN2,
+	COLLISION_GROUP_UNKNOWN3,
+	COLLISION_GROUP_NONE,			// Also known as "Default"

```

Comment:
Would it be better to rename it to COLLISION_GROUP_DEFAULT then?

Reply:
yeah I think I will move none to the first element and replace the current none with default